com.ugetdm.uGet.appdata.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.ugetdm.uGet</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPL-2.0</project_license>
+  <name>uGet</name>
+  <summary>Download manager using GTK+ and libcurl</summary>
```

Comment:
```suggestion
  <summary>Download manager</summary>
```
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines